BookReader/BookReader.css

```diff
@@ -508,6 +508,9 @@ body.BRfullscreenActive .mm-menu {
 .BRpagecontainer.BRemptypage {
```

Comment:
Yep, we don't include the build files (they get added automatically during the release flow). Trying to see if there's an easy git command to remove them :D

Reply:
This is about as close I can get :P `git rebase -X ours -x 'git checkout master BookReader && git status && git commit --amend --no-edit --allow-empty' master` , but it doesn't quite work. Oh well! This should do the trick

```sh
# Make sure master up-to-date
git checkout master
git pull origin master

# Go back to branch
git checkout text-selection

# Reset /BookReader
git checkout master BookReader
git add BookReader
git commit -m "Reset /BookReader"

# OPTIONAL: Rebase and combine all the commits into 1
# Or we can just select "Squash and Merge" in the GH UI.
git rebase -i master
```

Reply:
Ah, I didn't catch this one was a built file..